openwisp_ipam/api/views.py

```diff
@@ -137,10 +137,10 @@ def count(self):
         if self.stop is not None:
             return self.stop - self.start
         broadcast = int(self.subnet.subnet.broadcast_address)
-        # IPV4 (exclude broadcast)
         if self.subnet.subnet.max_prefixlen == 32:
+            if self.subnet.subnet._prefixlen == 32:
```

Comment:
```suggestion
            if self.subnet.subnet.prefixlen == 32:
```

`_prefixlen` is a private attribute. We should avoid using it. Use `prefixlen` instead. 
Also, add a comment here explaining why `/32` is handled separately.